lib/tiniergltf/base64.cpp

```diff
@@ -0,0 +1,112 @@
+/*
+base64.cpp and base64.h
```

Comment:
Duplicated file. It should be possible to remove `base64.cpp` so that the linker makes use of the functions in `MINETEST/src/util/base64.cpp`

This might only work when creating a statically linked library.

Reply:
It's slightly different actually: It returns a `vector<unsigned char>`, and the encoder has been removed. I'll try to reconcile it, it is indeed lazy to duplicate code like this.

As an aside: I really hate C++'s stdlib here. This should be an "array of chars" (with runtime-known size), but all C++ has to offer are vectors (which are basically "array lists") and strings (which we don't really want here but meh); a unique pointer isn't very feasible here because we'd need to pass the length around with it.

Reply:
Done in https://github.com/minetest/minetest/pull/14557/commits/b3e2ba919fa5eafb2bccc984b43c6d9ae72bfad3.